clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -5188,7 +5188,7 @@ def err_template_arg_not_object_or_func : Error<
 def err_template_arg_not_pointer_to_member_form : Error<
   "non-type template argument is not a pointer to member constant">;
 def err_template_arg_member_ptr_base_derived_not_supported : Error<
-  "sorry, non-type template argument of pointer-to-member type %1 that refers "
+  "non-type template argument of a pointer to member type %1, that refers "
```

Comment:
`pointer-to-member` is a term of art, we should keep it (and `a` is incorrect grammatically)

Reply:
Changed. I found it odd that there are places where it's sometimes 'pointer-to-member' and other times 'pointer to member' (like even right in the line above `5189: ...not a pointer to member...`)